packages/toolpad-components/package.json

```diff
@@ -38,7 +38,7 @@
     "url": "https://github.com/mui/mui-toolpad/issues"
   },
   "dependencies": {
-    "@mui/icons-material": "5.15.4",
```

Comment:
Did the script automatically update this?

Reply:
No, this was auto-generated by Renove since `icons-material` was republished